### PR TITLE
kata-deploy: Check that hub is installed

### LINKED ISSUE #728: Check hub installation
While trying to run `./release/kata-deploy-binaries.sh -p ....` it fails doing the push because it is missing `hub` https://github.com/github/hub. This issue is being found when you run this script for the first time.

### PR DESCRIPTION
**Pull request by GabyCT**
Running for the first time the kata-deploy script can fail if hub
is not installed it, this will avoid this issue.

Fixes #728

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: release/kata-deploy-binaries.sh [hunk 1]
```diff
@@ -67,6 +67,22 @@ EOT
...
 
+#Verify that hub is installed and in case that is not
+# install it to avoid issues when we try to push
+verify_hub() {
+	check_command=$(whereis hub | cut -d':' -f2)
+	# Install hub if is not installed
+	if [ -z ${check_command} ]; then
+		hub_repo="github.com/github/hub"
+		hub_url="https://${hub_repo}"
+		go get ${hub_repo}
```
Comment (devimc) [inline release/kata-deploy-binaries.sh]: missing `-d` and `|| true`? this can fail if `${GOPATH}/src/${hub_repo}` already exist .. ? :thinking:

Comment (GabyCT) [reply release/kata-deploy-binaries.sh]: @devimc changes applied

**Pull request merged**